src/App.tsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoId, setTodoId] = useState<number>(0);
+  const [loader, setLoader] = useState<boolean>(false);
+  const [filterByValue, setFilterByValue] = useState<string>('all');
+  const [filterByText, setFilterByText] = useState<string>('');
+
+  useEffect(() => {
+    getTodos()
+      .then(response => {
```

Comment:
Let’s use `async await` all over the code. It gives you a simpler way to work with asynchronous promise-based code